### PR TITLE
tests: update number of expected containers after migrating from nginx to traefik

### PR DESCRIPTION
**Pull request by kjaskiewiczz**

### CODE REVIEW

File: 07.Server-installation/03.Production-installation/docs.md [hunk 1]
```diff
@@ -831,9 +831,9 @@ git log --oneline master..HEAD
 
 <!-- Verification of Enterprise instance -->
```
Comment (lluiscampos) [inline 07.Server-installation/03.Production-installation/docs.md:833]: The Open Source test needs also an update, I assume. See lines 636-645.

**Review by lluiscampos — CHANGES_REQUESTED** [review]

**Comment by lluiscampos** [comment]
Also, don't you feel is about time to set these magic numbers as env variables at the top of the page(s)? :wink:

File: 07.Server-installation/03.Production-installation/docs.md (hunk 1 above)
Comment (kjaskiewiczz) [reply 07.Server-installation/03.Production-installation/docs.md:833]: right, fixed

**Comment by kjaskiewiczz** [comment]
> Also, don't you feel is about time to set these magic numbers as env variables at the top of the page(s)?

hmm, those are two different numbers in six places in two different files; I'm not sure if changing it to two different variables at the top of two different pages will help a lot; I'm not saying it will not help at all, but
IMO the main issue is that we're forgetting to update the numbers when changing the number of containers in our setup;
I don't know if that's possible, but maybe figuring out in the tests the number of expected containers from yaml files could be a solution?

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**